### PR TITLE
Improve error message "Could not find tools.jar" when running Gradle with a JRE

### PR DESCRIPTION
**Pull request by oreissig**
I stumbled over this message myself and didn't know what was wrong, and I have the issue pop up with  	colleagues now and then. I think giving a nudge in the direction of the most probable cause is a good idea.

### CODE REVIEW

File: subprojects/base-services/src/main/java/org/gradle/internal/jvm/JdkTools.java [hunk 1]
```diff
@@ -54,7 +54,9 @@ public static JdkTools current() {
         if (!javaVersion.isJava9Compatible()) {
             File toolsJar = javaInfo.getToolsJar();
             if (toolsJar == null) {
-                throw new IllegalStateException("Could not find tools.jar");
+                String javaHome = javaInfo.getJavaHome().getAbsolutePath();
```
Comment (eljobe) [inline subprojects/base-services/src/main/java/org/gradle/internal/jvm/JdkTools.java]: Just inline this call. We don't need the javaHome variable on any line other than the next one.

**Comment by oreissig** [comment]
I rebased the feature branch and did the minor correction requested.

**Comment by eljobe** [comment]
@oreissig Thanks for making this message clearer. Future Gradle users are already thanking you.

Contributions like this one make the Gradle codebase stronger over time,
~Pepper

**Comment by oreissig** [comment]
Thanks for merging :-)

**Pull request merged**